requirements/requirements.txt

```diff
@@ -7,7 +7,7 @@ httpx==0.27.0  # required by FastAPI
 Jinja2==3.1.3
 jsbeautifier==1.15.1
 kubernetes==29.0.0
-neon-mq-connector~=0.7
+neon-mq-connector==0.7.2a5
```

Comment:
`~=0.7,>=0.7.2a5` here would allow this to continue pulling newer compatible releases without explicit updates to dependencies

Reply:
> `~=0.7,>=0.7.2a5` here would allow this to continue pulling newer compatible releases without explicit updates to dependencies

Yeah, good point, but I think lets keep it strict until there will be next stable release